packages/react/src/components/DateTimePicker/__snapshots__/DateTimePicker.story.storyshot

```diff
@@ -2475,2479 +2475,3 @@ exports[`Storybook Snapshot tests and console checks Storyshots 1 - Watson IoT/D
-                >
-                  2018-10-27 19:34 to Now
-                </li>
-                <li
-                  className="bx--list__item iot--date-time-picker__listitem iot--date-time-picker__listitem--custom"
-                  onClick={[Function]}
-                  onKeyDown={[Function]}
-                  tabIndex={0}
-                >
-                  Custom Range
-                </li>
-                <li
-                  className="bx--list__item iot--date-time-picker__listitem iot--date-time-picker__listitem--preset"
-                  onClick={[Function]}
-                  onKeyDown={[Function]}
-                  tabIndex={0}
-                >
-                  Last 30 minutes
-                </li>
-                <li
-                  className="bx--list__item iot--date-time-picker__listitem iot--date-time-picker__listitem--preset"
-                  onClick={[Function]}
-                  onKeyDown={[Function]}
-                  tabIndex={0}
-                >
-                  Last 1 hour
-                </li>
-                <li
-                  className="bx--list__item iot--date-time-picker__listitem iot--date-time-picker__listitem--preset"
-                  onClick={[Function]}
-                  onKeyDown={[Function]}
-                  tabIndex={0}
-                >
-                  Last 6 hours
-                </li>
-                <li
-                  className="bx--list__item iot--date-time-picker__listitem iot--date-time-picker__listitem--preset iot--date-time-picker__listitem--preset-selected"
-                  onClick={[Function]}
-                  onKeyDown={[Function]}
-                  tabIndex={0}
-                >
-                  Last 12 hours
-                </li>
-                <li
-                  className="bx--list__item iot--date-time-picker__listitem iot--date-time-picker__listitem--preset"
-                  onClick={[Function]}
-                  onKeyDown={[Function]}
-                  tabIndex={0}
-                >
-                  Last 24 hours
-                </li>
-              </ol>
-            </div>
-          </div>
-          <div
-            className="iot--date-time-picker__menu-btn-set"
-          >
-            <button
-              aria-describedby={null}
-              aria-pressed={null}
-              className="iot--date-time-picker__menu-btn iot--date-time-picker__menu-btn-cancel bx--btn bx--btn--md bx--btn--secondary"
-              disabled={false}
-              onBlur={[Function]}
-              onClick={[Function]}
-              onFocus={[Function]}
-              onKeyUp={[Function]}
-              onMouseEnter={[Function]}
-              onMouseLeave={[Function]}
-              tabIndex={0}
-              type="button"
-            >
-              Cancel
-            </button>
-            <button
-              aria-describedby={null}
```

Comment:
why this got changed ?

Reply:
Not sure

Reply:
![image](https://user-images.githubusercontent.com/18572477/188666901-650c473e-b496-448f-8af9-b9a7839e7c3e.png)
Looks like space too. Its fine.